src/plots/cartesian/layout_attributes.js

```diff
@@ -897,7 +897,7 @@ module.exports = {
     },
     exponentformat: {
         valType: 'enumerated',
-        values: ['none', 'e', 'E', 'power', 'SI', 'B'],
+        values: ['none', 'e', 'E', 'power', 'SI', 'B', 'SI extended'],
```

Comment:
Let’s describe the difference between SI and SI extended in the description below, something like “SI uses prefixes from f (-15) to T (+12), switching to power notation beyond that range. SI extended uses prefixes from q (-30) to Q (+30).”